gnomad_constraint/constraint/constraint_pipeline.py

```diff
@@ -119,6 +137,12 @@ def main(args):
         help="Which dataset to use (one of gnomad, non_neuro, non_cancer, controls)",
         default="gnomad",
     )
+    parser.add_argument(
```

Comment:
I added this argument because: 1) I feel that population analysis is optional when we generate constraint table. 2) The downsampling in population has the potential risk to break the pipeline. instead of having an option for downsampling, I would make population analysis optional. 3) per population is going to be very underpowered. If we want this argument, I will update related code in other parts as well.